src/main/resources/db/migration/V1_2__create_media_request_table.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE IF NOT EXISTS media_request (
+  moj_mer_id                integer                     NOT NULL
```

Comment:
Do you need to add the CREATE SEQUENCE for this new table? `moj_mer_seq`
Also, the table name should be `moj_media_request` like in v23?